### PR TITLE
Runtasklistbugfixes

### PR DESCRIPTION
**Pull request by ritikaguptams**
Fix for the sequence of narration in 'Tasks'

fix for AI4D name property for confirm transfer pane

Fix for transfer page button availability

### CODE REVIEW

File: src/App/TaskListExecutionPage.xaml.cs [hunk 1]
```diff
@@ -34,6 +34,7 @@ public TaskListExecutionPage()
             this.NavigationCacheMode = NavigationCacheMode.Enabled;
             _selectedTaskList = -1;
             _selectedTaskListGuid = Guid.Empty;
+            _selectedTask = -1;
```
Comment (spaceisfun) [inline src/App/TaskListExecutionPage.xaml.cs:37]: I dont see any code that ever modifies _selectedTask. It remains -1 forever.

Comment (ritikaguptams) [reply src/App/TaskListExecutionPage.xaml.cs:37]: Noted, added the fix to consider the variable before making any selected index changes.

**Review by patelp3989 — APPROVED** [review]

**Pull request merged**